tests/drivers/uart/uart_elementary/README.txt

```diff
@@ -0,0 +1,13 @@
+The purpose of this test is to validate basic UART driver functions,
+that are not tested elsewhere.
+UART interrupt mode is used for the tests purpose.
+
+Hardware setup required for these tests:
+For single uart conviguration - UART0 TX connected to RX and CTS to RTS
+For dual uart configuratiom - UART0 and UART1 TXs and RXs cross-connected
```

Comment:
typo - `configuration`

Reply:
i think we can leave with it and get it fixed in upstream.